### ISSUE TITLE
TRAIN COPLENET

### ISSUE THREAD

**Issue by massimilianoporzio** [Project-MONAI/MONAI#1288]
Is it possible to train from scratch the CopleNEt? IS the noise robust dice loss already present in MONAI? The example of coplenet let the user to test it and do inference. How to use the NET and the LOSS in a novel training?

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Could you please help take a look at this question?
Thanks.

**Comment by massimilianoporzio** [comment]
It would be useful to have an example of the training as in the paper. Also to now what exactly have to be the input shape, if using patches during training etc. The code for copleNet I think have to be checked for the parameter "with_conv": 
self.up = UpSample(spatial_dims, in_channels1, in_channels2, scale_factor=2, with_conv=not bilinear)
In the MONAI 0.3.0 doc I did not find the 'with_conv' parameter.

**Comment by wyli** [comment]
thanks for the question, it'd be great to make the training code freely available, I don't have access to it for now.

for the documentation, please refer to the v0.3.0 version:
https://docs.monai.io/en/0.3.0/networks.html#monai.networks.blocks.UpSample

that's different from the current master branch:
https://docs.monai.io/en/latest/networks.html#monai.networks.blocks.UpSample
